R/exportToJson.R

```diff
@@ -20,22 +20,24 @@
 # @author Chris Knoll
 # @author Frank DeFalco
 
+
+# Run this definition of allReports when adding a new report
 # allReports <- c("CONDITION",
-#                "CONDITION_ERA",  
-#                "DASHBOARD", 
+#                "CONDITION_ERA",
+#                "DASHBOARD",
 #                "DATA_DENSITY",
 #                "DEATH",
 #                "DRUG",
 #                "DRUG_ERA",
 #                "HEEL",
 #                "OBSERVATION",
 #                "OBSERVATION_PERIOD",
-#                "PERSON", 
+#                "PERSON",
 #                "PROCEDURE",
 #                "VISIT",
 #                "MEASUREMENT",
 #                "META")
-# save(allReports,file="allReports.rda")
+# save(allReports,file="data/allReports.rda")

```

Comment:
Ahhh, I see, It was a code comment, not actually executed, OK, that's clear.  Thanks for fixing that, I think it's been wrong in that comment for a while .